source/index.html

```diff
@@ -2,11 +2,128 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Sedona</title>
   </head>
   <body>
+    <!-- Шапка сайта -->
+    <header class="hedaer">
+      <h1 class="header__title title-hidden">Sedona</h1>
+      <nav class="header__navigation">
+      <ul class="navigation-list navigation-list-left">
+        <li><a class="navigation-list__link">ГЛАВНАЯ</li>
+        <li><a class="navigation-list__link">ФОТО И ВИДЕО</li>
+      <ul class="navigation-list navigation-list-right">
+        <li><a class="navigation-list__link">ФОРМА ОТЗЫВА</li>
+        <li class="navigation-list__item"><a class="navigation-list__link">HTML ACADEMY</li>
+      </ul>
+      <img class="header__logo">
+      </nav>
+    </header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <!-- Основной контент страницы -->
+    <main>
+      <!-- Баннер главной страницы -->
+      <section class="section main-page-banner">
+        <h1>
+
+        </h1>
+      </section>
+
+      <!-- Преимущество Sedona -->
+      <section class="section benefits">
+        <div class="section__topic">
+        <h2 class="topic-headline">СЕДОНА — небольшой городок в АРИЗОНЕ, заслуживающий большего!</h2>
+        <p class="topic-text">Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
+        </div>
+
+      <!-- Первое преимущество -->
+      <section class="benefits__wrapper benefits__item--content-left ">
+        <div class="benefit">
+        <span class="benefit__headline--contrast">Настоящий городок</span>
```

Comment:
1. Я бы не стал модификатор писать каждому тексту блока, а задал бы родительскому элементу 1 раз и потом через него указал бы цвет всего текста внутри блока

2. Модификатор должен быть у элемента, в данном случае у тебя получается модификатор к несуществующему элементу
```suggestion
        <span class="benefit__headline benefit__headline--contrast">Настоящий городок</span>
```